matching-notification-migrations/src/main/java/org/phenotips/matchingnotification/internal/R74693PatientNetwork377DataMigration.java

```diff
@@ -90,7 +90,7 @@ public void hibernateMigrate() throws DataMigrationException, XWikiException
 
         try {
             Query q1 = session.createQuery("update " + PatientMatch.class.getCanonicalName()
-                + " set comments = concat('[{ comment:', comment, '}]') where comment <> ''");
+                + " set comments = concat('[{ \"comment\":\"', comment, '\"}]') where comment <> ''");
```

Comment:
The comment itself needs to be escaped before putting it in JSON. Quotes in the text can make the JSON invalid.